composer.json

```diff
@@ -26,12 +26,13 @@
         "source": "https://github.com/hedii/laravel-gelf-logger"
     },
     "require": {
-        "php": "^8.0",
-        "graylog2/gelf-php": "^1.7",
-        "illuminate/log": "^9.0"
+        "php": "^8.1",
+        "graylog2/gelf-php": "^2.0",
+        "illuminate/log": "^10.0"
     },
     "require-dev": {
-        "orchestra/testbench": "^7.0"
+        "orchestra/testbench": "^8.0",
+        "phpunit/phpunit": "^10.0"
```

Comment:
is there any reason you add phpunit 10 here? 

orchestra/testbench 8.0 already requires phpunit 10

Reply:
Yes, but they also allow using v9. So, I decided to lock to v10

https://github.com/orchestral/testbench/blob/8.x/composer.json#L26

Anyway I can remove it